2022/29xxx/CVE-2022-2929.json

```diff
@@ -0,0 +1,112 @@
+{
```

Comment:
Expected location (2022/2xxx/CVE-2022-2929.json) of file does not match actual location (2022/29xxx/CVE-2022-2929.json)